src/mots/config.py

```diff
@@ -201,7 +202,23 @@ def clean(file_config: FileConfig, write: bool = True):
     bmo_data = get_bmo_data(people)
     updated_people = People(people, bmo_data)
     logger.debug("Updating people configuration based on BMO data...")
-    file_config.config["people"] = updated_people.serialized
+
+    # If refresh is True, then we should do this. Otherwise, we should pick out
+    # pre-existing users and not touch those.
+    if refresh:
+        logger.debug("Refreshing all people...")
+        file_config.config["people"] = updated_people.serialized
+    else:
+        logger.warning("Only synchronizing new people with Bugzilla.")
+        updated_people_dict = {p["bmo_id"]: p for p in updated_people.serialized}
+        people_dict = {p["bmo_id"]: p for p in people}
+
+        additions = {
+            p: v for p, v in updated_people_dict.items() if "nick" not in people_dict[p]
```

Comment:
Rename all single letter variable names across the PR.

Reply:
I agree that this specific line is poorly named, it should say:

```suggestion
            bmo_id: person for bmo_id, person in updated_people_dict.items() if "nick" not in people_dict[bmo_id]
```

I don't agree that single letter variable names are universally unacceptable (e.g., if the scope is very small and the names are very obvious), but I'll go over the ones that are in this PR.

FWIW -- this particular section of the code was rewritten, and these instances were removed.

Reply:
The only acceptable use case is for something like `i` as an index when iterating over some enumerated collection. Beyond that a useful variable name is always better.

Reply:
> The only acceptable use case is for something like `i` as an index when iterating over some enumerated collection. Beyond that a useful variable name is always better.

I'd say there are other (albeit limited) instances where it could be acceptable, but not sure it is necessary to address that here. E.g., in certain limited scopes (e.g. dict and list comprehensions or very short for loops) and if it's clear what they represent. When assigning exceptions to variables (e.g. `e`, which is fairly accepted as a convention). When working with files --e.g. `f`. `i`, `j`, and `k` generally to represent indices. `k` and `v` instead of `key` and `value`. Probably a few others I missed.